development/prow-installer/pkg/cluster/wrapped.go

```diff
@@ -55,17 +56,36 @@ func (caw *APIWrapper) Create(ctx context.Context, clusterConfig Cluster) error
 			NodePools:      nodePools,
 		}}
 
-	createResponse, err := caw.ClusterService.Create(caw.ProjectID, clusterConfig.Location, ccRequest).Context(ctx).Do()
+	_, err := caw.ClusterService.Create(caw.ProjectID, clusterConfig.Location, ccRequest).Context(ctx).Do()
 	if err != nil {
-		return fmt.Errorf("couldn't create cluster: %w", err)
+		return "", fmt.Errorf("couldn't create cluster: %w", err)
+	} else {
+		log.Infof("successfully created cluster %s", clusterConfig.Name)
 	}
-	log.Printf("%#v\n", createResponse)
-	return nil
+
+	// get kubeconfig
+	kubeconfig := "./.kube/" + clusterConfig.Name + "_config"
```

Comment:
Maybe instead of hardcoding the path like this, we can use https://golang.org/pkg/os/#Getwd to be sure to always be in the working directory on every OS.

Reply:
It would be more convenient for sure. Will change that.